platform/nservicebus.include.md

```diff
@@ -1,3 +1,3 @@
-NServiceBus, the heart of the system, is a messaging and workflow framework that helps create distributed systems that are scalable, reliable and easy to modify. It supports various messaging patterns, handles long-running business processes in the form of [sagas](/nservicebus/sagas), and provides abstraction over multiple [queuing technologies](/transports/). While most queuing technologies try to make guarantees regarding 'at least once' or even 'exactly once' delivery, they often fall short of this promise. NServiceBus contains mechanisms to automatically solve intermittent delivery problems by retrying messages and falling back to an error queue where they can be exposed by the rest of the Platform for human intervention ([ServiceControl](/servicecontrol/), [ServicePulse](/servicepulse/)).
+NServiceBus, the heart of the system, is a messaging and workflow framework that helps create distributed systems that are scalable, reliable and easy to modify. It supports various messaging patterns, handles long-running business processes in the form of [sagas](/nservicebus/sagas), and provides an abstraction over several [queuing](/transports/) and [storage](/persistence/) technologies. While most queuing technologies try to make guarantees regarding 'at least once' or even 'exactly once' delivery, they often fall short of this promise. NServiceBus contains mechanisms to automatically solve intermittent delivery problems by retrying messages and falling back to an error queue where they can be exposed by the rest of the Platform for human intervention ([ServiceControl](/servicecontrol/), [ServicePulse](/servicepulse/)).
```

Comment:
I think this should have a paragraph break before "While most queuing technologies...". That part doesn't follow on from what precedes it.